guides/include-cleaner.md

```diff
@@ -25,13 +16,51 @@ This means:
 This is opinionated, and stricter than "will it compile". It implies adding
 `#includes` that the compiler does not strictly need. Some `#includes` that
 satisfy the compiler should be replaced with more specific ones.
+If your codebase does not broadly aim to follow this style, you should disable
+these features.
 
-There can be exceptions to these rules. Nevertheless, if your codebase does not
-broadly aim to follow this style, you should disable this feature.
+## Show which headers are used
+
+{:.v16}
+
+Hovering on a name will show you which header provides that symbol:
+![Symbol hover showing provider in VSCode](../screenshots/symbol_header.png)
+
+Hovering on an `#include` will show which of its symbols are used in the file.
+![#include hover showing symbols in VSCode](../screenshots/header_symbols.png)
+
+Using "find references" you can navigate to these uses.
```

Comment:
Not sure it's very clear what "using find references" means. It would be great to be slightly more specific, e.g., "Using the Find/Go to references feature of your editor/IDE", etc.

Reply:
Added "in your editor", this is definitely clearer.

I think we generally assume that people know that editor==IDE are synonyms and that features might be named slightly differently - spelling this out each time makes things pretty hard to read.

We could add a VSCode screenshot of this feature if it would help make it more concrete - I was just a bit reluctant to turn this into even more of a screenshot gallery.